source/features/fault.cc

```diff
@@ -340,6 +394,8 @@ namespace WorldBuilder
       // the starting radius is the distance from the bottom of the model to the surface.
       const double starting_radius = position_in_natural_coordinates.get_depth_coordinate() + depth - starting_depth;
 
+      BoundingBox<2> surface_bounding_box = get_bounding_box(position_in_natural_coordinates, depth);
```

Comment:
You can actually prevent this computing by putting it directly in the if statement. It is only used there, so that could save a lot of combination with the comment at the if statement.

Reply:
same for the other occurens